db/migrate/20220429092052_create_order_cycles_shipping_methods.rb

```diff
@@ -1,17 +1,16 @@
 class CreateOrderCycleShippingMethods < ActiveRecord::Migration[6.1]
   def up
-    create_table :order_cycle_shipping_methods do |t|
+    create_table :order_cycles_shipping_methods, id: false do |t|
       t.references :order_cycle
       t.references :shipping_method, foreign_key: { to_table: :spree_shipping_methods }
-      t.timestamps
     end
-    add_index :order_cycle_shipping_methods,
+    add_index :order_cycles_shipping_methods,
       [:order_cycle_id, :shipping_method_id],
-      name: "order_cycle_shipping_methods_join_index",
+      name: "order_cycles_shipping_methods_join_index",
```

Comment:
Do we need to give it a name?

Reply:
Yes, unfortunately it gives a `Index name 'index_order_cycles_shipping_methods_on_order_cycle_id_and_shipping_method_id' on table 'order_cycles_shipping_methods' is too long; the limit is 63 characters` error